apps/rush-lib/src/logic/VersionManager.ts

```diff
@@ -128,7 +128,7 @@ export class VersionManager {
     const lockStepVersionPolicyNames: Set<string> = new Set<string>();
 
     this._versionPolicyConfiguration.versionPolicies.forEach((versionPolicy) => {
-      if (versionPolicy instanceof LockStepVersionPolicy) {
+      if (versionPolicy instanceof LockStepVersionPolicy && versionPolicy.nextBump !== undefined) {
```

Comment:
This function should be renamed with this change. It's no longer just getting lockstepped projects, it's getting projects that aren't being driven by change types in changefiles.

Reply:
any suggestions on the name for that concept? maybe `manuallyVersionedProjects`?